### ISSUE TITLE
Write some evaluation tool that allows to detect hardest cases.

### ISSUE THREAD

**Issue by SebChw** [SebChw/Actually-Robust-Training#30]
This is idea mostly for the Sound Demixing wrong labels challenge. Knowing which example are hard we can suspect these are labeled wrongly. However, this may be also useful in simple training to see where we are poor. As a starting point try to do [this tutorial](https://pytorch.org/audio/main/tutorials/hybrid_demucs_tutorial.html)

**Comment by SebChw** [comment]
I calculated it using pretrained demucs. We can clearly see outliers for all instruments (I upload just one here). After this I filtered the examples with L1 error bigger than 0.6 and all of them was wrongly labeled basically I heard 3 cases:
* entirely wrong label (drums instead of vocals in the vocal track)
* Some parts missing (just background vocals and main missing)
* Some other tracks entirely within the class (bass + vocal in the vocal track) - NO BLEEDING entire track.
![output](https://user-images.githubusercontent.com/66476124/228746692-2d5fe165-ad9b-40db-b16f-572efd767ee4.png)

I'll ad the script for this calculation to the repo
